patch/armhf-sonic-kernel-config.patch

```diff
@@ -0,0 +1,97 @@
+From 3dd4829a62b201eb7091aaf31cc93634539e0ddd Mon Sep 17 00:00:00 2001
+From: Antony Rheneus <arheneus@marvell.com>
+Date: Wed, 25 Sep 2019 01:13:59 +0530
+Subject: [PATCH] armhf sonic kernel config
+
+---
+ debian/build/build_armhf_none_armmp/.config | 15 ++++++++++++++-
+ 1 file changed, 14 insertions(+), 1 deletion(-)
+
+diff --git a/debian/build/build_armhf_none_armmp/.config b/debian/build/build_armhf_none_armmp/.config
+index e2b7581..7229ca1 100644
+--- a/debian/build/build_armhf_none_armmp/.config
++++ b/debian/build/build_armhf_none_armmp/.config
+@@ -1501,6 +1501,7 @@ CONFIG_NET_ACT_POLICE=m
+ CONFIG_NET_ACT_GACT=m
+ CONFIG_GACT_PROB=y
+ CONFIG_NET_ACT_MIRRED=m
++CONFIG_NET_ACT_SAMPLE=m
+ CONFIG_NET_ACT_IPT=m
+ CONFIG_NET_ACT_NAT=m
+ CONFIG_NET_ACT_PEDIT=m
+@@ -1717,6 +1718,7 @@ CONFIG_NFC_PN533=m
+ CONFIG_NFC_PN533_USB=m
+ # CONFIG_NFC_PN533_I2C is not set
+ # CONFIG_NFC_ST95HF is not set
++CONFIG_PSAMPLE=y
+ CONFIG_LWTUNNEL=y
+ CONFIG_DST_CACHE=y
+ CONFIG_NET_DEVLINK=m
+@@ -2011,6 +2013,8 @@ CONFIG_EEPROM_LEGACY=m
+ CONFIG_EEPROM_MAX6875=m
+ CONFIG_EEPROM_93CX6=m
+ # CONFIG_EEPROM_93XX46 is not set
++CONFIG_EEPROM_SFF_8436=y
++CONFIG_EEPROM_OPTOE=y
+ CONFIG_CB710_CORE=m
+ # CONFIG_CB710_DEBUG is not set
+ CONFIG_CB710_DEBUG_ASSUMPTIONS=y
+@@ -3386,6 +3390,7 @@ CONFIG_I2C_MUX_PCA954x=m
+ # CONFIG_I2C_MUX_PINCTRL is not set
+ # CONFIG_I2C_MUX_REG is not set
+ # CONFIG_I2C_DEMUX_PINCTRL is not set
++CONFIG_I2C_MUX_MLXCPLD=y
+ CONFIG_I2C_HELPER_AUTO=y
+ CONFIG_I2C_ALGOBIT=y
+ CONFIG_I2C_ALGOPCA=m
+@@ -3819,9 +3824,11 @@ CONFIG_SENSORS_MAX1668=m
+ CONFIG_SENSORS_MAX6639=m
+ CONFIG_SENSORS_MAX6642=m
+ CONFIG_SENSORS_MAX6650=m
++CONFIG_SENSORS_MAX6620=y
+ # CONFIG_SENSORS_MAX6697 is not set
+ # CONFIG_SENSORS_MAX31790 is not set
+ # CONFIG_SENSORS_MCP3021 is not set
++CONFIG_SENSORS_MLXREG_FAN=y
+ CONFIG_SENSORS_ADCXX=m
+ # CONFIG_SENSORS_LM63 is not set
+ CONFIG_SENSORS_LM70=m
+@@ -3953,6 +3960,7 @@ CONFIG_DA9052_WATCHDOG=m
+ # CONFIG_GPIO_WATCHDOG is not set
+ # CONFIG_XILINX_WATCHDOG is not set
+ # CONFIG_ZIIRAVE_WATCHDOG is not set
++CONFIG_MLX_WDT=y
+ CONFIG_ARM_SP805_WATCHDOG=m
+ # CONFIG_CADENCE_WATCHDOG is not set
+ CONFIG_HAVE_S3C2410_WATCHDOG=y
+@@ -6032,6 +6040,8 @@ CONFIG_LEDS_TCA6507=m
+ #
+ # CONFIG_LEDS_BLINKM is not set
+ # CONFIG_LEDS_SYSCON is not set
++CONFIG_LEDS_MLXREG=y
++CONFIG_LEDS_USER=y
+ 
+ #
+ # LED Triggers
+@@ -6435,6 +6445,9 @@ CONFIG_SPEAKUP_SYNTH_DUMMY=m
+ CONFIG_CHROME_PLATFORMS=y
+ # CONFIG_CROS_EC_CHARDEV is not set
+ CONFIG_CROS_EC_PROTO=y
++CONFIG_MELLANOX_PLATFORM=y
++CONFIG_MLXREG_HOTPLUG=y
++CONFIG_MLXREG_IO=y
+ CONFIG_CLKDEV_LOOKUP=y
+ CONFIG_HAVE_CLK_PREPARE=y
+ CONFIG_COMMON_CLK=y
+@@ -7011,7 +7024,7 @@ CONFIG_RAS=y
+ # Android
+ #
+ # CONFIG_ANDROID is not set
+-CONFIG_NVMEM=m
++CONFIG_NVMEM=y
```

Comment:
why do you change the y? we prefer to use m if not absolutely neccessary. This applies all kernel configuration.